ruby/red-arrow/ext/arrow/converters.hpp

```diff
@@ -761,8 +773,6 @@ namespace red_arrow {
     VISIT(Time32)
     VISIT(Time64)
     VISIT(Timestamp)
-    // TODO
-    // VISIT(Interval)
```

Comment:
Ah, we need to implement dictionary array cases with tests.
Could you revert this change and open a new JIRA issue for adding tests of `Arrow::DictionaryArray#values`

Reply:
@kou 
sure!
I opened two JIRA issue!

https://issues.apache.org/jira/browse/ARROW-15935
https://issues.apache.org/jira/browse/ARROW-15936